griptape/engines/extraction/csv_extraction_engine.py

```diff
@@ -20,6 +20,9 @@ class CsvExtractionEngine(BaseExtractionEngine):
     column_names: list[str] = field(default=Factory(list), kw_only=True)
     system_template_generator: J2 = field(default=Factory(lambda: J2("engines/extraction/csv/system.j2")), kw_only=True)
     user_template_generator: J2 = field(default=Factory(lambda: J2("engines/extraction/csv/user.j2")), kw_only=True)
+    formatter_fn: Callable[[Any], str] = field(
+        default=lambda value: "\n".join(f"{key}: {val}" for key, val in value.items()), kw_only=True
+    )
 
```

Comment:
it is not immediately obvious that a dictionary will be passed to this function. 
should it be renamed to make it more clear? or move the `dict(zip(...` logic to the default formatter?

Reply:
Type hint should've been Callable[[dict], str], updated